src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetSourcesService.cs

```diff
@@ -51,6 +53,24 @@ public ValueTask<IReadOnlyList<PackageSourceContextInfo>> GetPackageSourcesAsync
                 .ToList());
         }
 
+        public IReadOnlyList<SourceRepository> GetAuditSources()
```

Comment:
nit: I think `GetEnabledAuditSources` is a better method name here, since the method returns only audit sources that are enabled.

```suggestion
        public IReadOnlyList<SourceRepository> GetEnabledAuditSources()
```

Reply:
I agree that's clearer! This was just copied from `src\NuGet.Clients\NuGet.CommandLine\Commands\RestoreCommand.cs` which is named that way. I probably won't rename Restore's command in this PR.

Reply:
Done!